workshop/Lab5/README.md

```diff
@@ -0,0 +1,227 @@
+```console
+$ svcat get classes
+     NAME             DESCRIPTION             UUID     
++------------+---------------------------+------------+
+  mariadb      Helm Chart for mariadb      mariadb     
+  mongodb      Helm Chart for mongodb      mongodb     
+  mysql        Helm Chart for mysql        mysql       
+  postgresql   Helm Chart for postgresql   postgresql  
+  redis        Helm Chart for redis        redis    
+```
+
+The same can be done for the Plans of those Classes by running `svcat get plans`. Note that
+this response contains a bit more info, so it can display quite a lot of text.
+```console
+$ svcat get plans
+   NAME       CLASS               DESCRIPTION                   UUID        
++---------+------------+--------------------------------+------------------+
+.
+.
+.
+  3-2-9     redis        Open source, advanced            redis-3-2-9
+                         key-value store. It is
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+  4-0-2     redis        Open source, advanced            redis-4-0-2
+                         key-value store. It is
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+  4-0-6     redis        Open source, advanced            redis-4-0-6
+                         key-value store. It is
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+  4-0-7     redis        Open source, advanced            redis-4-0-7
+                         key-value store. It i
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+  4-0-8     redis        Open source, advanced            redis-4-0-8
+                         key-value store. It is
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+  4-0-9     redis        Open source, advanced            redis-4-0-9
+                         key-value store. It is
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+```
+
+The numbered tuples are the version of the helm chart minibroker uses to deploy the services we ask it to provison.
```

Comment:
hmmm, so while minibroker is using the version of Redis as the name, I wouldn't explain it that way. Stick with talking about that column being "the name". I really don't think people put the version of the service as the name very often - I know bluemix doesn't.  I think minbroker is a bit odd in this respect because for the most part users should be unaware of which version of the service they're using (not all the time, but usually)